### PR TITLE
Toggle updates

### PR DESCRIPTION
**Pull request by philschanely**
## Description

This PR does a little tidying on toggles such as Switch, Radio, and Checkbox:

- Adds `message` elements to Radio and Checkbox (Rails)
- Fixes border colors for various states of Radio to match specs

Also this PR adds two utilities for adding `card-` and `panel-`-level grid settings on an element, such as when you need to group a set of elements within a panel (such as within a form or for conditional rendering): `.sage-panel-grid` and `sage-card-grid`. Basically everything to build a panel or card container minus the padding and border treatments.

## Test notes

See Radio, Switch, and Toggle for stability.

### Estimated impact

1. (LOW) Updates to Radio, Switch, and Toggle to include a message/hint and correct state coloring. No adverse effects expected. Investigate any Sage-based forms such as the following for continued expected behavior:
   - [ ] Blog post editor form (radio for publishing on the right sidebar)
   - [ ] Navigation editor, edit a link modal (checkbox for whether or not to be only visible to logged in users)

### CODE REVIEW

File: docs/app/views/examples/elements/checkbox/_preview.html.erb [hunk 1]
```diff
@@ -9,6 +9,24 @@
...
+    id:"c1b",
+    label_text: "Checkbox",
+    checked: false,
+    disabled: false,
+    has_error: false,
+    message: 'This is where a "hint" message would appear.',
+  } %>
+
+  <%= sage_component SageCheckbox, {
+    id:"c1b",
```
Comment (pixelflips) [inline docs/app/views/examples/elements/checkbox/_preview.html.erb:22]: Duplicate IDs with line 13 above.

File: packages/sage-assets/lib/stylesheets/core/_variables.scss [hunk 2]
```diff
@@ -63,7 +63,7 @@ $sage-radio-colors: (
   checked: sage-color(primary),
   checked-disabled: sage-color(primary, 200),
   checked-inner: sage-color(white),
-  default: sage-color(grey, 4000),
+  default: sage-color(grey, 400),
```
Comment (pixelflips) [inline packages/sage-assets/lib/stylesheets/core/_variables.scss:66]: Nice catch!

**Review by pixelflips — APPROVED** [review]
👍🏼  - Found one small issue w/ IDs in documentation, but this LGTM! Nice work!

**Review by joedakroub — APPROVED** [review]
No concerns with these changes other than what Phillip pointed out about the duplicate IDs. 👍🏼

**Pull request merged**